esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -4753,7 +4753,7 @@ bool CWsWorkunitsEx::onWUCreateZAPInfo(IEspContext &context, IEspWUCreateZAPInfo
         zipFileNameWithPath.append(zipFolder).append(zipFileName.str());
         pathNameStr.set(folderToZIP.str()).append("/*");
 
-        const char* password = req.getPassword();
```

Comment:
You can't remove this line, it still has to work for old clients.

Do something like:

```
const char* password = req.getZAPPassword();
if (!password || !*password)
    password = req.getPassword();
if (password && *password)
    zipCommand.appendf("zip -j --password %s %s %s", password, zipFileNameWithPath.str(), pathNameStr.str());
else
```

Reply:
Changed